docs_user/modules/proc_migrating-mon-from-controller-nodes-config-target-nodes.adoc

```diff
@@ -93,14 +115,55 @@ Ensure that Ceph Monitors are in a quorum and listed by the `ceph orch` command:
 ----
 +
 ----
-[ceph: root@oc0-controller-0 /]# ceph orch host ls
+$ sudo cephadm shell -- ceph orch host ls
 HOST              ADDR           LABELS          STATUS
-oc0-ceph-0        192.168.24.14  osd mon _admin
-oc0-ceph-1        192.168.24.7   osd mon _admin
-oc0-ceph-2        192.168.24.8   osd mon _admin
-oc0-controller-0  192.168.24.15  _admin mgr mon
-oc0-controller-1  192.168.24.23  _admin mgr mon
-oc0-controller-2  192.168.24.13  _admin mgr mon
+ceph-0        192.168.24.14  osd mon mgr _admin
+ceph-1        192.168.24.7   osd mon mgr _admin
+ceph-2        192.168.24.8   osd mon mgr _admin
+controller-0  192.168.24.15  _admin mgr mon
+controller-1  192.168.24.23  _admin mgr mon
+controller-2  192.168.24.13  _admin mgr mon
+----
+
+Setup a Ceph client on the first controller node that is used during the rest
+of the procedure to interact with Ceph. Setup an additional IP address on the
+storage network that is used to interact with Ceph when the first controller
+node is decommissioned.
```

Comment:
```suggestion
. Set up a Ceph client on the first Controller node that is used during the rest
of the procedure to interact with {Ceph}. Set up an additional IP address on the
storage network that is used to interact with {Ceph} when the first Controller
node is decommissioned:
```

Reply:
@fmount I formatted this text as a step and the rest as substeps. The reason is because this text was interfering with the order of the steps. The steps' order started over after this text.